src/main/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidator.java

```diff
@@ -30,6 +35,87 @@ public boolean isValid(Object value, ConstraintValidatorContext context) {
         timeEntry.getActualEndTime());
 
     session.setHibernateFlushMode(FlushMode.AUTO);
-    return timeEntryClashes.isEmpty();
+    if (!timeEntryClashes.isEmpty()) {
+      ClashingProperty clashingProperty = getClashingProperty(timeEntry, timeEntryClashes);
```

Comment:
Even though we fully expect there to be a clashing property given the query has detected it, the `getClashingProperty` method still has the potential to return null, so we can technically encounter a `NPE` on ln 51.
Maybe we need to do a null check somewhere here.

Reply:
I've changed this to default to start and end time clash. I think this results in the best behaviour on the front end if we somehow get a clash but can't identify the clashing property, unless we want to just display the generic service error?

Reply:
Yeah looks good to me, if someone has an issue with it in the future under the rare circumstance that it happens, we can change as necessary.